test/arkane/ess/arkaneGaussianTest.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python3
+
+###############################################################################
+#                                                                             #
+# RMG - Reaction Mechanism Generator                                          #
+#                                                                             #
+# Copyright (c) 2002-2021 Prof. William H. Green (whgreen@mit.edu),           #
```

Comment:
I don't remember if we have an official rule on when we update the dates at the top of each file, but just thought I’d point out that it’s 2 years old now. Probably worth a separate PR in the future

Reply:
I am pro removing anyway, it feels very 70's :laughing:

Reply:
It's part of the "new release" workflow; I think there's a script to do it.